app/lib/stale.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Stale
+  # The API responder performs conditional GET and only produces the response for stale objects,
+  # but calling `stale?` for controllers that deal with objects that are not ActiveRecord descendants fails
+  # because it relies on `updated_at` field which is not available
+  # In such cases the object will always be considered stale
+  def stale?(object = nil, **freshness_kwargs)
+    has_updated_at = object.respond_to?(:updated_at) || object&.all? { _1.respond_to? :updated_at }
+    return super if has_updated_at
+
+    true
```

Comment:
Another minor nitpick :angel: 
```suggestion
    !has_updated_at || super
```

Reply:
Hmm... I see what you did there, but to me it's much harder to understand that intention of the method if written the way you suggest... 
I personally would sacrifice code style for understandability in this case. WDYT @jlledom ?

Reply:
It's about being used to use boolean expressions instead of conditionals + boolean literals.

Either way, in this case logic is exactly the same with both types of expression, so not super important to change anything. I leave it up to your discretion.

Reply:
I agree with @mayorova. If I found that suggestion I would have needed a few moments to really understand what it does.

Reply:
I'm keeping it as it is for now